### PR TITLE
Ofir/extend tool for lb lab

### PR DESCRIPTION
**Pull request by ofir-lb**
Adding 'generic' node/server option, support for only single-node 4 to 12 ssds, no chrony, 
option to get data-ips conf for single data interface
Also added centos ga 3-3-x option (w.o kernel base)
Also fix docker ansible execution: using 'sudo docker run -i' instead of 'sudo docker run -it"

### CODE REVIEW

**Comment by felixm-lb** [comment]
Awesome - only request I have is to move the jq install into the InstallInstallerSoftware method to match the rest of the codebase and it'll be good to go. Love the changes!

File: install_lightbits.sh [hunk 1]
```diff
@@ -1,8 +1,25 @@
...
+# Updates:
+# --------
+# 07-Aug-2023 [OE]   added 'generic' node/server option, support now only single-node 4 to 12 ssds, no chrony, data-ips conf for single data interface
+#                    added some echo prints (ease debug)
+#                    added install for jq (was missing on some vanila's)
+#                    added centos ga 3-3-x option (w.o kernel base)
+#                    fix docker ansible execution: using 'sudo docker run -i' instead of 'sudo docker run -it"
+#                    TODO: to consider add support for dual nodes data interfaces
+INSTALL_LIGHTBITS_VERSION="V1.01"
+yum install jq -y
```
Comment (felixm-lb) [inline install_lightbits.sh:13]: @ofir-lb here's the line I'd move into the InstallInstallerSoftware method

**Comment by felixm-lb** [comment]
Aside from the single comment I think it's good to go - really appreciate the work Ofir! Once the jq install is moved I'll merge.

**Comment by ofir-lb** [comment]
@felixm-lb putting jq  insall under InstallInstallerSoftware is not good enough as you also use jq in previous checks...
so I will move it and push another PR to fix it
Cheers!

**Pull request merged**